plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -46,12 +48,27 @@ const uint16_t EuroliteProFactory::VENDOR_ID_MK2 = 0x0403;
 
 const char EuroliteProFactory::ENABLE_EUROLITE_MK2_KEY[] =
     "enable_eurolite_mk2";
+const char EuroliteProFactory::EUROLITE_MK2_SERIAL_KEY[] =
+    "eurolite_mk2_serial";
 
 EuroliteProFactory::EuroliteProFactory(ola::usb::LibUsbAdaptor *adaptor,
                                        Preferences *preferences)
   : BaseWidgetFactory<class EurolitePro>("EuroliteProFactory"),
     m_adaptor(adaptor),
     m_enable_eurolite_mk2(IsEuroliteMk2Enabled(preferences)) {
+  const std::vector<std::string> serials =
+    preferences->GetMultipleValue(EUROLITE_MK2_SERIAL_KEY);
```

Comment:
Indent further as its not an if statement or similar
```suggestion
      preferences->GetMultipleValue(EUROLITE_MK2_SERIAL_KEY);
```

Reply:
Done thanks